packages/rich-text/src/plugins/EmbeddedEntityBlock/index.tsx

```diff
@@ -108,18 +106,7 @@ export function getWithEmbeddedEntityEvents(
     return function handleEvent(event: KeyboardEvent) {
       if (!editor) return;
 
-      const isEnter = event.keyCode === 13;
-      const [, pathToSelectedElement] = getNodeEntryFromSelection(editor, nodeType);
-
-      if (pathToSelectedElement) {
-        if (isEnter) {
-          event.preventDefault();
-          moveToTheNextLine(editor);
-        } else if (event.key === 'Backspace') {
-          event.preventDefault();
-          Transforms.removeNodes(editor, { at: pathToSelectedElement });
-        }
-      } else if (
```

Comment:
We don't need this anymore since we are using the `InsertBeforeFirstVoidBlock` plugin.